src/js/rx/components/Prescription.jsx

```diff
@@ -63,14 +57,15 @@ class Prescription extends React.Component {
           <div className="rx-prescription-number">
             Rx #: {attrs['prescription-id']}
```

Comment:
While you're at it, this can also change to "Prescription #" like the mocks. Been meaning to make that change.